### DISCUSSION TITLE
How to make responsive web for login Ui for quasar

### DISCUSSION THREAD

**Discussion by sahdan96** [quasarframework/quasar#8883] [CLI - PWA mode]
I use Quasar framework and Vue js to make a login ui. So there problem is when switch to mobile, the Ui did not resize to fit. And I still new to quasar and javascript.

    <template>
      <div class="q-pa-md" style="width: 400px">
        <q-img src="./../assets/logo.png" width="100%">
          <div class="row justify-center bg-transparent" style="width: 100%">
            <div class="column justify-center items-center q-pa-md bg-banner">
              <q-img src="./../assets/logo.png" width="100px" />
                <div class="text-black text-subtitle1">My Project</div>
            </div>
          </div>
        </q-img>
        <q-form class="q-mt-md">
          <q-input filled label="Your email *" lazy-rules />

          <q-input filled label="Password *"> </q-input>

          <div style="margin-top: 40px" class="q-gutter-x-sm">
            <q-btn label="Login" color="primary" />
            <q-btn flat label="Sign Up" />
            <q-btn flat label="Forgot Password?" />
          </div>
        </q-form>
      </div>
    </template>

    <script>
    </script>
    <style scoped>
    .bg-banner {
      background-color: white;
      opacity: 0.85;
      border-radius: 3px;
    }
    </style>

**Comment by hawkeye64** [comment]
@sahdan96 What do you mean `switch to mobile`? My guess is you are using the browser's dev tools to emulate a mobile device? If this is the case, then you have to refresh the page any time you change the emulated device so Quasar gets the proper information (device type, etc).

**Reply by sahdan96** [reply]
I mean using the browser 'responsive design mode'.

**Reply by hawkeye64** [reply]
You are setting the width to 400px
```
<div class="q-pa-md" style="width: 400px">
```
Maybe use the breakpoint classes

**Reply by sahdan96** [reply]
How do I use the breakpoint classes. Can you add the code?

**Reply by hawkeye64** [reply]
Look at this page: https://quasar.dev/layout/grid/row
